modules/RTC/TraceablePeerConnection.js

```diff
@@ -1797,6 +1806,71 @@ TraceablePeerConnection.prototype.setRemoteDescription = function(
 };
 
 /**
+ * Inserts an H264 payload into the description if not already present.
+ *
+ * @param {RTCSessionDescription} description - An RTCSessionDescription
+ * to inject with an H264 payload.
+ * @returns {RTCSessionDescription}
+ */
+TraceablePeerConnection.prototype._injectH264IfNotPresent = function(
+        description) {
+    const parsedSdp = transform.parse(description.sdp);
+    const videoMLine = parsedSdp.media.find(m => m.type === 'video');
+
+    if (!videoMLine) {
+        logger.debug('No videoMLine found, no need tp inject H264.');
+
+        return description;
+    }
+
+    if (videoMLine.rtp.some(rtp => rtp.codec.toLowerCase() === 'h264')) {
+        logger.debug('H264 codec found in video mLine, no need to inject.');
+
+        return description;
+    }
+
+    const { fmtp, payloads, rtp } = videoMLine;
+    const payloadsArray = payloads.toString().split(' ');
+    let dummyPayloadType;
+
+    for (let i = 127; i >= 96; i--) {
```

Comment:
i can be const.

Reply:
I don't think variable i can be const in this case because then variable i cannot be reassigned a value in the increment expression.

Reply:
You're right, of course!